pixeltable/catalog/globals.py

```diff
@@ -52,23 +52,23 @@ def validated(cls, name: str, error_prefix: str) -> MediaValidation:
             raise excs.Error(f'{error_prefix} must be one of: [{val_strs}]')
 
 class IfExistsParam(enum.Enum):
+    ERROR = 0
+    IGNORE = 1
+    REPLACE = 2
+    REPLACE_FORCE = 3
+    '''
     ERROR = 'error'
```

Comment:
This is not a useful comment.

Reply:
Yes, was left by mistake. Removed.
<!-- Reviewable comment -OEr21rL4FWO7IACodkp:b-470wdb -->
<!-- Sent from Reviewable.io -->